routers/web/web.go

```diff
@@ -642,6 +642,12 @@ func RegisterRoutes(m *web.Route) {
 			m.Post("/teams/{team}/edit", bindIgnErr(forms.CreateTeamForm{}), org.EditTeamPost)
 			m.Post("/teams/{team}/delete", org.DeleteTeam)
 
+			m.Group("/times", func() {
+				m.Get("/by_repos", org.TimesByRepos)
+				m.Get("/by_members", org.TimesByMembers)
+				m.Get("/by_milestones", org.TimesByMilestones)
+			}, context.OrgAssignment(true, true))
```

Comment:
```suggestion
			}, context.OrgAssignment(false, true))
```
Only org admin should be able to see it right?